tests/testthat/test-simList.R

```diff
@@ -14,7 +14,8 @@ test_that("simList object initializes correctly", {
   w <- getOption("width")
   options(width = 100L)
   out <- utils::capture.output(show(mySim))
-  expect_equal(length(out), 75)
+  nline <- if (out[60]=="NULL") 75 else 73 # data.table v1.11.0 no longer prints "NULL" data.table (bug fix 8 in NEWS)
```

Comment:
maybe tag which version of `NEWS` for future reference

Reply:
The comment on that line includes v1.11.0 ?